service/build.gradle

```diff
@@ -132,21 +132,21 @@ dependencyUpdates.resolutionStrategy = {
 }
 
 // https://jeremylong.github.io/DependencyCheck/dependency-check-gradle/configuration.html
-dependencyCheck {
-  // Specifies if the build should be failed if a CVSS score above a specified level is identified.
-  // range of 0-10 fails the build, anything greater and it doesn't fail the build
-  failBuildOnCVSS = System.getProperty('dependencyCheck.failBuild') == 'true' ? 0 : 11
-  suppressionFile = 'config/owasp/suppressions.xml'
-  analyzers {
-    // Disable scanning of .NET related binaries
-    assemblyEnabled = false
-  }
-  analyzedTypes = ['jar']
-  scanConfigurations = configurations.findAll {
-    !it.name.startsWith('checkstyle') && !it.name.startsWith('jacoco') && !it.name.startsWith('integrationTest') &&
-      !it.name.startsWith('smokeTest') && !it.name.startsWith('functionalTest')
-  }.collect { it.name }
-}
+//dependencyCheck {
+//  // Specifies if the build should be failed if a CVSS score above a specified level is identified.
```

Comment:
instead of commenting out, could we skip it?
`dependencyCheck {
  skip = true
...
}
`

Reply:
perfect hint, thanks:)